### ISSUE TITLE
Domains: human readable values (display name / comment)

### ISSUE THREAD

**Issue by m-kuhn** [opengisch/projectgenerator#47]
In the UML editor it's possible to define a comment for each value in a domain. This is added to the .ili file

E.g.

```
Domain = (
  /** Explication with spaces and Umlauts etc.
   */
   Value
)
```

This seems not to be imported by ili2pg yet.

**Comment by gacarrillor** [comment]
See also https://github.com/claeis/ili2db/issues/88, where a `@dispName` meta-attribute is planned for adding a display name for each domain value.

**Comment by gacarrillor** [comment]
See also https://github.com/claeis/ili2db/issues/88, where `@dispName` meta-attribute is planned for adding a display name for each domain value.

**Comment by gacarrillor** [comment]
FYI, the `@ili2dbdispName` meta-attribute is now implemented in ili2pg v3.9.1. Additionally, comments (ilidocs) in the .ili file are written into a new `description` column in ENUM tables.

So, this:

```
DOMAIN 
   myDomain = (
      !!@ili2db.dispName="Display name for value 1"
      /** Explication with spaces and Umlauts etc.
      */
      Value1
   )

```
Is transferred to the database in this way:

**mydomain (TABLE)**

```
itfcode | ilicode | seq | inactive |        dispname          |   description
0       | Value1  |     | FALSE    | Display name for value 1 | Explication with spaces and Umlauts etc.
```

**Comment by m-kuhn** [comment]
Fixed in https://github.com/opengisch/projectgenerator/commit/770e568b931c5bf898998a5605ec2200f9de383b ( @gacarrillor please reopen if you think there's anything left to do)
